### PR TITLE
Update linting, bump deps

### PR DESCRIPTION
**Pull request by rwwagner90**

### CODE REVIEW

**Comment by gilest** [comment]
@rwwagner90 I think [this LinkComponent import](https://github.com/ember-learn/ember-cli-addon-docs/blob/master/addon/components/docs-link.js#L2) will need [ember-legacy-built-in-components](https://github.com/emberjs/ember-legacy-built-in-components).

**Comment by rwwagner90** [comment]
@gilest would you perhaps be interested in opening a PR for that?

**Comment by gilest** [comment]
> @gilest would you perhaps be interested in opening a PR for that?

Sure thing it's #1015 😄

**Comment by rwwagner90** [comment]
@gilest it seems like we still have the same failures on beta and canary. Any ideas?

**Comment by gilest** [comment]
> @gilest it seems like we still have the same failures on beta and canary. Any ideas?

Looks like you got some PRs that fixed it. Thanks for the release – my CI is green again 🙌🏻

**Comment by rwwagner90** [comment]
> Looks like you got some PRs that fixed it. Thanks for the release – my CI is green again 🙌🏻

Glad to hear it! My builds are still not green somehow 😂

File: babel.config.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* eslint-env node */
```
Comment (SergeAstapov) [inline babel.config.js:1]: @rwwagner90 don't want to be show stopper, from https://github.com/babel/ember-cli-babel/issues/418 I got impression that addons should not ship `babel.config.js`

**Pull request merged**